docs/client-side/angular/sample/src/Controllers/Api/PersonController.cs

```diff
@@ -0,0 +1,22 @@
+﻿using AngularSample.Models;
+using Microsoft.AspNet.Mvc;
+using System.Collections.Generic;
+
+namespace AngularSample.Controllers.Api
+{
+    public class PersonController : Controller
+    {
+        [Route("/api/people")]
+        public JsonResult GetPersons()
```

Comment:
GetPersons -> GetPeople
Return the Person list directly instead of returning JsonResult to enable support for content negotiation.